src/index.js

```diff
@@ -257,7 +257,7 @@ Debug.extend(exported, {isSafari, getPerformanceMetrics: PerformanceUtils.getPer
  *
  * @function setRTLTextPlugin
  * @param {string} pluginURL URL pointing to the Mapbox RTL text plugin source.
- * @param {Function} callback Called with an error argument if there is an error.
+ * @param {Function} callback Called with an error argument if there is an error, or a null error argument on success.
```

Comment:
I think it would be better to say `, or no arguments if the plugin loads successfully`